gradle-plugins/build.gradle

```diff
@@ -5,6 +5,11 @@
 plugins {
     // Support convention plugins written in Groovy. Convention plugins are build scripts in 'src/main' that automatically become available as plugins in the main build.
     id 'groovy-gradle-plugin'
+    id 'org.jetbrains.intellij' version '1.14.1'
```

Comment:
@greene858 Should this line be: `id 'org.jetbrains.intellij-plugin' version '1.14.1'`?
RE: [Slack](https://dsva.slack.com/archives/C04QLHM9LR0/p1688053889336159)